index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en-GB" >
+<head>
+	
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Damian HOW</title>
+	<link rel="stylesheet" href="styles.css" type="text/css" />
+	<link rel="stylesheet" href="normalize.css" type="text/css" />
+	<link href="https://fonts.googleapis.com/css2?family=Azeret+Mono&display=swap" rel="stylesheet" />
+	
+</head>
+
+<body>
+	<div id="page-container" >
+		<div id="content-wrap" >
+			<header>
+				<div class="logo" >
+					<img src="images/logofinal.jpg" alt="Logo" />
+				</div>
+				<div class="title" >
+					<h3>Damian's House of Wonders</h3>
+				</div>
+				<div class="hidden" >
+					<img src="images/rsz_logo.jpg" alt="Hidden logo" />
```

Comment:
whats the intended functionality associated with this? will this image ever be seen?

Reply:
This was the invisible image I talked to you about that I hide at the right hand side of the screen at the top in the header so that I can use justify-content: space between; to keep the actual logo in the top left but also have the heading in the middle